lib/notifier.js

```diff
@@ -25,18 +27,29 @@ function notifyIfChanged(businesses) {
 }
 
 function notifyConsole(message, options){
+    let date_ob = new Date();
```

Comment:
1. Use `const` when you don't reassign.
2. Use camelCasing for variable names.
3. Don't include type in variable names.
4. Give descriptive names (a new Date gives the current date/time, so the name should reflect that).
```
const now = new Date();
```